src/test/javascript/InstructorStudentListJsTest.js

```diff
@@ -64,9 +64,10 @@ test('filterSection()', function(){
     equal($("#studentsection-c0\\.0").is(':visible'), true, "All sections should be visible");
     equal($("#studentsection-c0\\.1").is(':visible'), true, "All sections should be visible");
     equal($("#studentsection-c1\\.0").is(':visible'), true, "All sections should be visible");
-    equal($("#section_check-0-0").is(':checked'), true, "Course 2 Section A should be re-selected");
     equal($("#section_check-0-1").is(':checked'), true, "Course 2 Section B should be re-selected");
-    equal($("#section_check-1-0").is(':checked'), true, "Course 3 Section C should be re-selected");
```

Comment:
Okay, would make sense if it was checking that it won't be affected, but its not doing that.